.github/workflows/chromatic.yml

```diff
@@ -24,12 +24,15 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '16'
-          cache: 'npm'
+
+      - name: Install pnpm
+        run: npm install pnpm -g
+
```

Comment:
nope!
use the correct action, which should be executed before npm install, for caching pnpm modules